index.html

```diff
@@ -41,8 +41,6 @@ <h1 class="img-responsive appium-logo">Appium</h1>
     </div>
 </div>
 
-{% include appiumconf-banner.html %}
```

Comment:
why not keep everything and just comment this out? when we do the same thing (hopefully) next year, it'd be nice not to have to re-do the plumbing

Reply:
Yeah, that's not a bad idea. We can just swap in new images.